Python/notify_me/README.md

```diff
@@ -0,0 +1,40 @@
+# Notify Me
+
+* supported OS: windows 10
+* Push desktop notification every 4 hours to notify you the TODO items
+
+
+
+### Pre-requisites
+1. install python in your windows 10
+2. use pip to install plyer
+
+```
+$ pip install plyer
+```
+
+### How to start
+
+Double click notify-me.bat to start the program
+
+### How to run every time system boot
+1. Create a short-cut of notify-me.bat under notify_me dirctory
+2. press win + r and enter "shell:startup":
+  ![run](https://lh3.googleusercontent.com/1FL17gW0BCTsxSOWMOg6B40J-jZOK7Q1mc745ZTBRePsfgT3RT9x3xA32deC4hmwjiUh9Kxe-3eNCdQFVoVR90kOgCG11IxilIRuCvWTcKG6RIz-DI7ahGmENoCj1Ontt5IT1_AkXpWMBoHfLng4U7N5KXVt_R76gFAAFyMC6fT7MNEWJh5T7ET4WFy0tfve6mONr1SbNInfuu9ImH0nKtpsBcEHJn-KkbqinKjIyFVMIw0Emw5ASWNYMrfjgkLv-dmtNMUnMFl55H3mxvK4oAZhEPgccWrfuEw3-dc3PeDRcyjYvEMr_Ymn5qzL-QzbN2kNLHubeO44a2DNgiL_mlVdxEIUgNxNiQXe167qAIU2AhufeLKv8cJeJvdohZjJwMrZd3zB7iyHSmzVQQZs0VLGa02AW1VD0gieCK37y3wy5LLmFEe_5FvsKajR6Gyb2XZ9kfSHdDqaeQ8yq1quzDwitq9Q_2FNqqISNuyvYewaZ3Y-YfnnwEoaXbU0PDHR7xOM1zeqCUxXUNzdfLnat2_43EVkeqTxALUy7g3_8rhiGdCJDRk3zHZxYUMHYEahTMvbe624yrmmUMlW-hNmYhWMYVVhY-BikjuagvF4DDQjrNZUHzNHx8gSHuFrKt4eJgcYSHJEmsPYv99GSPQDtaigTk-74KbB3wjxBnBiqiIdkPKTb7s4T1TaQiTWDnA=w824-h478-no?authuser=0)
+3. move the "notify-me.bat - shortcut" into the startup program directory:
+  ![startup folder](https://lh3.googleusercontent.com/YWlwzc5PS06FBjeLbKvVnoBUJr_TN8oIXCWyJ5pb7yGOeeZzxCLoj-3KIsGzrcsH-vn82-KNSRELfNW5aT8IypdVWL186ryrPHogSTkeHypUcKSjRYtq9mv1j_2GWwWZVlbvtUeFlT82Mic2qz1yIYU7rXM1QU98CkxG7QWWrS5Pk3UDPYE0DgXWEmgnYScdLLzY-eykrFbGDBvTR0OsFtdZNnxu-pC65zVoiZ3pRL3LVrGR_YDyWq70ur0_oZ6v0yV-ObyEgQBzoRtZBdY3VsOTexpe0ypPR21Ad9OKMK1FIKsnbNnmnxPKiHAeNVGaAXvR2Zil2p2Oqj5EpkCMLbOoq0FPrviYksDrzmCwAvs9FEzHXydkS_1k3FhQHNqIfS0lGnL6rJZma7-gXJO99Hhpte9J4Y05iMH-qVTMesFhCy87I3juYbQkpKI67iWqa1Ee2KFIO9oPqot7seo4iNc6v3IrxNUt0pR9qmMkMftX3W5TPp1wwAEEq5UOdSyzMEf3yO7xPHICsR5FChp9IUkiIRUGTxMotGXjDWL9QaM0mJ5EkbxGWOEkB6734Aqg-Hu_4PE-v3Lxu5fqN4y8Ps9TIYtiSN-7FHKciEYviA9cFROPCLFdUJ2vzlRkXiWSH1I1MTWdLRNUMb1EjGD5HfXYljJF1JDFkwLgmKIWb_q_ja9RB7hcOcRsIOwK-g4=w1218-h294-no?authuser=0)
+
+### How to use
+
+1. After execute notify-me.bat, you will notice there is a "notify-me.txt" on your desktop.
+2. Add your todo list into the notify-me.txt 
+  - One line for one item
+  - You could use "#"(hash) mark to make comment
+3. Every 4 hours, your PC will push the desktop notification for current todo items in the notify-me.txt
+
+## Thanks and Enjoy
+
```

Comment:
Remove these trailing lines

Reply:
I had removed those trailing lines.
Please kindly help to double check if it is ok.